app/src/main/java/xyz/jhughes/laundry/ModelOperations.java

```diff
@@ -49,31 +49,4 @@ public int compare(Location l1, Location l2) {
         });
         return locations;
     }
-
-    public static Integer[] getAvailableCounts(List<Machine> machines){
```

Comment:
Why did this get deleted?

Reply:
@piticent123  This method was really hard to read. It returned 4 different "Machine Counts", two for washers, two for dryers. I moved this logic to the Location model in the LaundryParser folder. That should make this method **way** easier to maintain. There will be a lot of small improvements like this while we attempt to separate the logic currently in the app. Hope this clears things up

Reply:
:+1: to be honest when i do code reviews i mainly look for bad practice type stuff or obvious bugs because im assuming if it compiles on your computer all the functionality is probably still there, so i was just curious where this function went.